Sources/Spezi/Spezi.docc/Spezi Ecosystem.md

```diff
@@ -0,0 +1,268 @@
+
+This module is included and demonstrated in the **[Spezi Template Application](https://github.com/StanfordSpezi/SpeziTemplateApplication)**.
+
+@Row{
+    @Column {
+        ![Screenshot displaying the account setup view with an email and password prompt and a Sign In with Apple button.](https://raw.githubusercontent.com/StanfordSpezi/SpeziAccount/main/Sources/SpeziAccount/SpeziAccount.docc/Resources/AccountSetup.png)
+    }
+    @Column {
+        ![Screenshot displaying the Signup Form for Account setup.](https://raw.githubusercontent.com/StanfordSpezi/SpeziAccount/main/Sources/SpeziAccount/SpeziAccount.docc/Resources/SignupForm.png)
+    }
+    @Column {
+        ![Screenshot displaying the Account Overview.](https://raw.githubusercontent.com/StanfordSpezi/SpeziAccount/main/Sources/SpeziAccount/SpeziAccount.docc/Resources/AccountOverview.png)
+    }
+}
+
+> Tip: Check out the [SpeziAccount Documentation](https://swiftpackageindex.com/stanfordspezi/speziaccount/documentation/speziaccount) for implementation details.
+
+
+### Spezi Contact
+- The **[SpeziContact](https://github.com/StanfordSpezi/SpeziContact)** module facilitates the presentation and management of contact information, offering customizable display options for contact details including names, phone numbers, and email addresses. 
+
+This module is included and demonstrated in the **[Spezi Template Application](https://github.com/StanfordSpezi/SpeziTemplateApplication)**.
+
+@Row{
+    @Column {
+       ![Screenshot showing contact display](https://raw.githubusercontent.com/StanfordSpezi/SpeziContact/refs/heads/main/Sources/SpeziContact/SpeziContact.docc/Resources/Overview.png)
+    }
+    @Column {
+       > Tip: Check out the [SpeziContact Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziContact/documentation/spezicontact) for implementation details.
+    }   
+    @Column {
+       ![ ]()
+    }  
+}
+
+
+### Spezi AccessGuard
+- The **[SpeziAccessGuard](https://github.com/StanfordSpezi/SpeziAccessGuard)** module provides robust security features for SwiftUI views, including numeric access code protection, biometric authentication support, and customizable timeout settings for automatic locking.
+
+@Row {
+    @Column {
+        ![Screenshot showing access code entry](https://raw.githubusercontent.com/StanfordSpezi/SpeziAccessGuard/refs/heads/main/Sources/SpeziAccessGuard/SpeziAccessGuard.docc/Resources/AccessGuarded.png)
+    }
+    @Column {
+        ![Screenshot showing biometric auth](https://raw.githubusercontent.com/StanfordSpezi/SpeziAccessGuard/refs/heads/main/Sources/SpeziAccessGuard/SpeziAccessGuard.docc/Resources/AccessGuarded-Biometrics.png)
+    }
+    @Column {
+      > Tip: Check out the [SpeziAccessGuard Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziAccessGuard/documentation/speziaccessguard) for implementation details.
+    }  
+}
+
+
+### Connected Devices and Spezi Bluetooth
```

Comment:
I get why you combined both modules into one heading, it makes sense to bundle it. However, I think we need a larger decision on how to structure and sort the documentation in general (see comments above).